### PR TITLE
mobile menu for portfolio with javascript Dom

### PR DESCRIPTION
**Pull request by FrankMugagga**
I used javascript  DOM event listener to design my mobile portfolio menu
The menu is responsive in a way that onclick of the links they take you to the sections of the portfolio
and onclick of the hamburger, it opens and on close it disappears

### CODE REVIEW

**Review by osamaashraf6 — COMMENTED** [review]
Hi @FrankMugagga 😊, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🆘💀

You have submitted a project, that : 
1. PR description and PR Title are both not descriptive and do not reflect the tasks for this milestone.
2. Not all linters check are set for this project. I can not see the webhint linter, so kindly 😊follow these steps for all linter  [THESE](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
3.  Kindly make sure placeholders are replaced with your project and personal data. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌
.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Describe between 1-3 key features of the application.
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-Add a link to your deployed project.
+Add a link to your deployed project
 
 - [Live Demo Link](https://frankmugagga.github.io/Mugagga_Frank_Portfolio/)
```
Comment (hiromataba) [inline README.md:92]: - [ ] Please use the correct stack for this section here ,
You used, HTML,CSS and Javascript and replace the ones actually there

image

File: README.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ Describe between 1-3 key features of the application.
```
Comment (hiromataba) [inline README.md:86]: - [ ] Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @FrankMugagga     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @FrankMugagga,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Davidosky007 — COMMENTED** [review]
Hi @FrankMugagga 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with the wrong PR since this is a new milestone `Portfolio: details popup window` Kindly create a new PR for this 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**